omod/src/test/resources/config.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC "-//OpenMRS//DTD OpenMRS Config 1.0//EN" "http://resources.openmrs.org/doctype/config-1.6.dtd">
+
+<module configVersion="1.6">
+
+    <id>bahmni-events</id>
+    <name>Bahmni Events</name>
+    <version>${project.parent.version}</version>
+    <description>Provides bahmni events</description>
+    <package>org.bahmni.module.events</package>
+    <author>Bahmni</author>
+
+    <activator>org.bahmni.module.events.EventsActivator</activator>
+
+    <require_version>1.11.6</require_version>
+
+    <messages>
+        <lang>en</lang>
```

Comment:
Do we need this for each language. If so, then Bahmni supports a lot more languages right now. If this is a backend only module, then maybe there is nothing to translate. If so, you only need english.

Reply:
Removed the messages